### PR TITLE
Fix piranha plant hitboxes

### PR DESCRIPTION
**Pull request by Sunketchupm**
Bug reported in #150

### CODE REVIEW

File: src/game/behaviors/piranha_plant.inc.c [hunk 1]
```diff
@@ -122,6 +122,10 @@ void piranha_plant_act_woken_up(void) {
      */
     o->oDamageOrCoinValue = 3;
 #endif
+    // The piranha plant still normally has the larger hitbox up to this point
+    cur_obj_set_hitbox_radius_and_height(250.0f, 200.0f);
+    cur_obj_set_hurtbox_radius_and_height(150.0f, 100.0f);
```
Comment (Isaac0-dev) [inline src/game/behaviors/piranha_plant.inc.c]: why is this necessary? the original sm64 decomp didn't have this, so how come its necessary to force the hitbox? that'll mess with mods and its not vanilla behavior.
if this is the best work around for this bug, thats fine, I just wonder why the bug even exists in the first place

Comment (Sunketchupm) [reply src/game/behaviors/piranha_plant.inc.c]: In vanilla sm64, the hitbox gets set while sleeping and that doesn't change until the plant starts biting.
In coop, the biting hitbox is set every frame. I don't know why it does this, but I assume it's to force a hitbox to exist no matter what. Regardless, since the biting hitbox is set every frame, when the action becomes start waking up, it will instead use the biting hitbox rather than the sleeping hitbox.

Comment (Isaac0-dev) [reply src/game/behaviors/piranha_plant.inc.c]: I'd suggest its a bug in itself. This has been a common issue with the behaviors in coop, where certain parts of their behaviors are always as though there is always a player nearby. This is just slightly broken logic that can be fixed

Comment (Sunketchupm) [reply src/game/behaviors/piranha_plant.inc.c]: The hitbox being set to biting every frame isn't a bug. It's set directly in the loop code.

Comment (Isaac0-dev) [reply src/game/behaviors/piranha_plant.inc.c]: yeah i know what you mean i'm saying there's probably a bug in the code that was changed for syncing the piranha plants

**Review by Isaac0-dev — APPROVED** [review]
